src/output.rs

```diff
@@ -1,39 +1,77 @@
-use std::collections::HashMap;
-
-use frost_ed25519 as frost;
-
-use frost::keys::{KeyPackage, PublicKeyPackage};
+use frost::keys::{PublicKeyPackage, SecretShare, VerifiableSecretSharingCommitment};
 use frost::Identifier;
+use frost_ed25519 as frost;
 use itertools::Itertools;
+use std::collections::HashMap;
 
 pub trait Logger {
     fn log(&mut self, value: String);
 }
 
+fn encode_commitment(vss_commitment: VerifiableSecretSharingCommitment) -> String {
+    let serialized = vss_commitment.serialize();
+    let num = serialized.len().to_string();
+
+    let mut out = hex::encode(num);
```

Comment:
This will only work up to 9 shares, you could encode the length as a byte instead. Something like this (untested):

```
let num = serialized.len();
let mut out = hex::encode(&[num]);
```